helm-charts/hbase-chart/templates/meta/_initnnscript.tpl

```diff
@@ -1,6 +1,7 @@
 {{- define "hbasecluster.initnnscript" }}
+{{/* init containers which are required one time only during bootstrap  define their isBootstrap flag as true */}}
 - name: init-namenode
-  isBootstrap: {{ default false .isBootstrap }}
+  isBootstrap: true
```

Comment:
`isBootstrap: {{ default true .isBootstrap }}`

Reply:
`isBootstrap: {{ default true .isBootstrap }}`

it would be wrong , in case of parent isBootstrap false , it will again bypass the conditions . As it will pick from parent isBootstrap flag everytime if isBootstrap flag is specified .

Logic  is build  on similar lines like other init containers like `_dnslookup.tpl` , `_faultdomain.tpl` and `_refreshnn.tpl` which are required everytime have already their isBootstrap set to false in the code like following 

```
{{- define "hbasecluster.faultdomain" }}
- name: init-faultdomain
  isBootstrap: false
  command:
  - /bin/bash
```

Similarly the init containers required one time should be defined as true

Reply:
got it